### DISCUSSION TITLE
Does mongoose create additional db connections

### DISCUSSION THREAD

**Discussion by skrishnanzetwerk** [Automattic/mongoose#11031] [Q&A]
When I create a connection using `mongoose.createConnection` and specify a `minPoolSize:5`, will only 5 db connections be created by mongoose. Or does mongoose create any additional connections for internal use?

**Comment by vkarpov15** [comment]
The MongoDB driver may create additional connections. If you want to limit the max number of connections, set `maxPoolSize` along with `minPoolSize`

**Reply by skrishnanzetwerk** [reply]
Thank you. So if I use createConnection with `minPoolSize:5` and `maxPoolSize:10`, at application start only 5 db connections will be made. And depending on slow queries and application load a maximum of 10 connections will be made to the db. Is my understanding correct?

**Reply by vkarpov15** [reply]
@skrishnanzetwerk that's correct.
